src/Sulu/Bundle/SecurityBundle/Tests/Functional/Command/CreateUserCommandTest.php

```diff
@@ -18,10 +18,8 @@
 
 class CreateUserCommandTest extends SuluTestCase
 {
-    /**
-     * @var CommandTester
```

Comment:
Hmm. most of the time I personally don't even bother defining member properties in tests. I think they should be as small as possible. If you have a test that has 20 member properties then thats 80 extra lines that you have to scroll past before you see the tests.